### PR TITLE
Added histogram metrics to tasks

### PR DESCRIPTION
**Pull request by alimez**
Each task will have 4 related metrics (_sum, _bucket, _count, _created). 
More details on what each metric does here: https://prometheus.io/docs/practices/histograms/

### CODE REVIEW

**Review by aarontp — APPROVED** [review]
LGTM, just one small nit.

**Comment by hacktobeer** [comment]
LGTM but is there a way to bucket the runtime per task type or is this metric calculated over all tasks without splitting it up per type? (eg runtimes for PlasoTask, StringTask etc)

**Review by hacktobeer — APPROVED** [review]

File: turbinia/workers/__init__.py [hunk 1]
```diff
@@ -766,52 +767,57 @@ def run_wrapper(self, evidence):
...
-        # Check if Task's job is available for the worker.
-        active_jobs = list(job_manager.JobsManager.GetJobNames())
-        if self.job_name.lower() not in active_jobs:
-          message = (
-              'Task will not run due to the job: {0:s} being disabled '
-              'on the worker.'.format(self.job_name))
-          self.result.log(message, level=logging.ERROR)
-          self.result.status = message
-          return self.result.serialize()
+      task_runtime_metrics = Histogram(
```
[the remark this answers is not in the document]
Comment (alimez) [reply turbinia/workers/__init__.py:770]: By default, it creates different buckets for us, we can also specify our buckets later on. Example:

```
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="7.5"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="5.0"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="2.5"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="10.0"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="1.0"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.75"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.5"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.25"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.1"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.075"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.05"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.025"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.01"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="0.005"}
✔FinalizeRequestTask_duration_seconds_bucket{instance="turbinia-worker-2ae18fc5f45cf923-1:8000",job="turbinia-gcp_node",le="+Inf"}
```

File: turbinia/workers/__init__.py [hunk 2]
```diff
@@ -766,52 +767,57 @@ def run_wrapper(self, evidence):
...
+          '{}_duration_seconds'.format(self.name), 'Seconds to run {}'.format(
```
[the remark this answers is not in the document]
Comment (alimez) [reply turbinia/workers/__init__.py]: done.

**Comment by aarontp** [comment]
LGTM.  I think @hacktobeer's comment was answered out of band (yes, we'll have per-task histograms), so I'm going to merge this.

**Pull request merged**